typings/index.d.ts

```diff
@@ -979,6 +1013,8 @@ declare module 'discord.js' {
     public readonly roles: GuildMemberRoleManager;
     public user: User;
     public readonly voice: VoiceState;
+    public avatarURL(options?: ImageURLOptions & { dynamic?: boolean }): string | null;
+    public displayAvatarURL(options?: ImageURLOptions & { dynamic?: boolean }): string;
```

Comment:
```suggestion
    public avatarURL(options?: ImageURLOptions): string | null;
    public displayAvatarURL(options?: ImageURLOptions): string;
```
This isn't needed after #5805

Reply:
Will fix
Btw are the types Snowflake | number correct for user and guildIds? I see they're used on the Avatar function so I just went with them but it doesn't make much sense right?